README.md

```diff
@@ -59,7 +59,7 @@ For now referenced here until we have a more official doc: https://github.com/no
 - `all`: All node versions
 - `lts_active`: Head of LTS and Active major version lines
 - `lts`: Head of current LTS lines
-- `active`: The newest version in the active but not maintenence mode lines
+- `active`: The newest version in the active but not maintenance mode lines
```

Comment:
This is fixed in multiple unrelated PRs.  Obviously it is correct, but also this will cause conflicts when those others land.  I guess it will be a race to who lands first!

Reply:
If it’s the exact same change, there should not be a conflict :)

Reply:
I think multiple adjacent lines as a chunk will right? https://github.com/pkgjs/nv/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R62